tutorials/navigation/navigation_introduction_3d.rst

```diff
@@ -34,16 +34,28 @@ Godot provides the following objects and classes for 3D navigation:
         - NavRegion RID
             Reference to a specific navigation region that can hold navigation mesh data.
             The region can be enabled / disabled or the use restricted with a navigationlayer bitmask.
+        - NavLink RID
+            Reference to a specific navigation link that connects two navigation mesh positions over arbitrary distances.
         - NavAgent RID
             Reference to a specific avoidance agent with a radius value use solely in avoidance.
 
 The following SceneTree Nodes are available as helpers to work with the NavigationServer3D API.
 
 - :ref:`NavigationRegion3D<class_NavigationRegion3D>` Node
     A Node that holds a Navigation Mesh resource that defines a navigation mesh for the NavigationServer3D.
-    The region can be enabled / disabled.
-    The use in pathfinding can be further restricted through the navigationlayers bitmask.
-    Regions can join their navigation meshes by proximity for a combined navigation mesh.
+
+    - The region can be enabled / disabled.
+    - The use in pathfinding can be further restricted through the navigationlayers bitmask.
+    - Regions can join their navigation meshes by proximity for a combined navigation mesh.
+
+- :ref:`NavigationLink3D<class_NavigationLink3D>` Node
+    A Node that connects two positions on navigation mesh over arbitrary distances for pathfinding.
+
+    - The link can be enabled / disabled.
+    - The link can be made one-way or bidirectional.
+    - The use in pathfinding can be further restricted through the navigationlayers bitmask.
+
+    Links tell the pathfinding that a connection exists and at what cost. The actual agent handling and movement needs to happen in custom scripts.
```

Comment:
I'm a bit confused why the last sentence is here on the 3D and 2D page. Is it implying that there's extra stuff that needs to be done in a pathfinding script for navigation links to work? Or does it mean moving agents around in general? Cause if it's the latter then it should be under the NavigationAgent node description.

Reply:
You need to use a script to properly handle movement through a link because it is just a virtual connection between two points. There is nothing specific to NavigationAgent, it is the same for whatever custom setup users might use.

Reply:
Got it, thanks for clarifying!